### PR TITLE
Return of the audible emote cooldown.

### PR DESCRIPTION
**Pull request by Rohesie**
Instead of the old 30 seconds cooldown, a softer 5 seconds.

### CODE REVIEW

**Comment by F-OS** [comment]
Jesus fuck thank you

**Comment by Rohesie** [comment]
How would you implement it without generating another set of spam for the admins?

**Comment by Rohesie** [comment]
Honestly I have other fires to extinguish. Most of my PRs recently are bug, exploit and balance fixes. If you want to add extra admin tools, feel free to. I'd rather focus on the essential now.

**Comment by F-OS** [comment]
Alright, thanks anyway!

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -73,10 +73,13 @@
 				m_type = 2
 
 		if ("clap")
-			if (!is_mob_restrained())
-				message = "<B>[comm_paygrade][src]</B> claps."
-				m_type = 2
-				playsound(src.loc, 'sound/misc/clap.ogg', 25, 0)
+			if (is_mob_restrained())
+				return
+			if(audio_emote_cooldown(player_caused))
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/human/emote.dm]: this one can be combined with the above if statement

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -149,10 +152,13 @@
 				message = "<B>[comm_paygrade][src]</B> glares."
 
 		if ("golfclap")
-			if (!is_mob_restrained())
-				message = "<B>[comm_paygrade][src]</B> claps, clearly unimpressed."
-				m_type = 2
-				playsound(src.loc, 'sound/misc/golfclap.ogg', 25, 0)
+			if (is_mob_restrained())
+				return
+			if(audio_emote_cooldown(player_caused))
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/human/emote.dm]: ditto

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -250,6 +259,10 @@
 			m_type = 1
 
 		if ("pain")
+			if (muzzled)
+				return
+			if(audio_emote_cooldown(player_caused))
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/human/emote.dm]: ditto

File: code/modules/mob/living/carbon/human/emote.dm [hunk 4]
```diff
@@ -277,6 +290,10 @@
 			m_type = 1
 
 		if("scream")
+			if (muzzled)
+				return
+			if(audio_emote_cooldown(player_caused))
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/human/emote.dm]: ditto

**Pull request merged**